### DISCUSSION TITLE
计算器合法性问题

### DISCUSSION THREAD

**Discussion by huwenjing0819** [liusy58/NJU_AdvancedProgramming_SP22#58] [Q&A] [answered]
请问计算器中出现满足括号匹配的空括号对可以认为是不合法的吗？比如，()3+4认定为不合法表达式。

**Comment by binbin00** [comment, marked as answer]
算，在此感谢大家想出的一些边边角角的不合法表达式，这说明大家考虑得很周全，我们也鼓励大家这么做。
但是，我们测试的时候不会特地找一些边边角角的不合法表达式去坑大家的，会着重测常见的几种，比如ppt里面列出来的。
